tst/EnergyPlus/unit/WeatherManager.unit.cc

```diff
@@ -688,7 +688,6 @@ TEST_F(EnergyPlusFixture, ASHRAE_Tau2017ModelTest)
 
     bool ErrorsFound(false);
     state->dataEnvrn->TotDesDays = 2;
-    state->dataGlobal->NumOfTimeStepInHour = 0;
```

Comment:
NumOfTimeStepInHour is now 0 again because GetProjectData has not been called (i.e.,, I removed init_state from process_idf on this pass).